Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .js.coffee assets and views
 gem 'coffee-rails', '~> 4.0.0'
 # See https://github.com/sstephenson/execjs#readme for more supported runtimes
-#gem 'therubyracer',  platforms: :ruby
```

Comment:
For heroku deployment I had commented the therubyracer gem but we need that gem for IT deployment.